openapi/components/schemas/PaymentTokens/DigitalWalletValidation/ApplePayValidation.yaml

```
@@ -8,10 +8,24 @@ allOf:
       type:
         type: string
       validationRequest:
-        description: The validation request provided by the Apple Pay SDK to validate payment ability
+        description: The validation request.
         type: object
+        properties:
+          validationURL:
```

I just noticed this, should we be using cameCase?
```suggestion
          validationUrl:
```

---

I decided to use the variable name as in the Apple Pay SDK event. It will be compatible in backed.